components/pagination/index.jsx

```diff
@@ -0,0 +1,36 @@
+import { createElement } from "complate-stream";
+
+export function Pagination({ "aria-label": label, class: additionalClass, size }, ...children) {
+	let klass = ["pagination"];
+	if(additionalClass) {
+		klass.push(additionalClass);
+	}
+	if(size) {
+		klass.push(`pagination-${size}`);
+	}
+	return <nav aria-label={label}>
+		<ul class={klass.join(" ")}>
+			{children}
+		</ul>
+	</nav>;
+}
+
+export function PageItem({ active, disabled, href }, ...children) {
+	let tag ="span";
```

Comment:
There is a space missing here, and it is hurting me.

Reply:
Whoops!